src/js/Options.js

```diff
@@ -6,18 +6,13 @@ const options = {
 
 };
 
-let admOptionValue = document.getElementById("local").value;
```

Comment:
(I originaly suggested to use `function admSelectCheck(nameSelect) {` in this line. To workaround CSP, this function is written in another way.)